gobject-subclass/src/properties.rs

```diff
@@ -102,6 +106,14 @@ pub enum Property<'a> {
         fn() -> glib::Type,
         PropertyMutability,
     ),
+    Variant(
```

Comment:
We need to add support for `Variant`s in `Value`s in GLib for this first, you only added `VariantType` so far :)

Reply:
I think that's already there though, because of ```glib_wrapper```.

Reply:
Indeed, but it's doing it wrong unfortunately. The macro considers it a boxed type but it isn't anymore: `Note: GLib 2.24 did include a boxed type with this name. It was replaced with this fundamental type in 2.26.`

Reply:
https://github.com/gtk-rs/glib/issues/344